ax3_mixins/mixins.py

```diff
@@ -4,10 +4,12 @@
 from django.shortcuts import redirect
 from django.views.generic import TemplateView
 
+from .utils import is_ajax
+
 
 class AjaxRequestMixin:
     def dispatch(self, request, *args, **kwargs):
-        if not request.is_ajax():
+        if not is_ajax(request):
```

Comment:
Change to: `if request.META.get('HTTP_X_REQUESTED_WITH') != 'XMLHttpRequest'`

Reply:
Done